api/client/retl/types.go

```diff
@@ -57,6 +57,7 @@ type RETLSourceCreateRequest struct {
 	SourceType           SourceType         `json:"sourceType"`
 	SourceDefinitionName string             `json:"sourceDefinitionName"`
 	AccountID            string             `json:"accountId"`
+	IsEnabled            bool               `json:"enabled"`
```

Comment:
Enabled would be more pragmatic of a Key here, what do you think ?
```suggestion
	Enabled            bool               `json:"enabled"`
```